contents/blog/the-posthog-array-1-43-0.md

```diff
@@ -0,0 +1,103 @@
+---
+date: 2023-01-16
+title: "Array 1.43.0: Massive performance improvements!"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? Check out [our roadmap](/roadmap) to see what we're working on and what new beta features are available! You can also subscribe to [our Hogmail newsletter](/newsletter)!
+
+## Update guide for self-hosted users
+
+Any notes about async migrations, etc. go here.
```

Comment:
@tiina303 Just tagging you here, as you've provided info on this for the last few Arrays